framework/helpers/BaseInflector.php

```diff
@@ -410,6 +410,18 @@ public static function id2camel($id, $separator = '-')
         return str_replace(' ', '', StringHelper::mb_ucwords(str_replace($separator, ' ', $id), self::encoding()));
     }
 
+    /**
+     * Converts a CamelCase action name into an ID in lowercase.
+     * Words in the ID are concatenated using the specified character '-'.
+     * For example, 'CreateUser' will be converted to 'create-user'.
+     * @param string $name the string to be converted
+     * @return string the resulting ID
+     */
+    public static function camel2idAction($name)
```

Comment:
As @rob006 already noted, this method could be moved to the help command itself. It's hardly reusable.